evap/staff/tests/test_live.py

```diff
@@ -36,16 +37,16 @@ def test_changes_form_data(self):
             evaluation=evaluation,
             contributor=responsible,
             order=0,
-            role=Contribution.Role.EDITOR,
-            textanswer_visibility=Contribution.TextAnswerVisibility.GENERAL_TEXTANSWERS,
+            role=Contribution.Role.CONTRIBUTOR,
+            textanswer_visibility=Contribution.TextAnswerVisibility.OWN_TEXTANSWERS,
```

Comment:
I changed this so that the values selected in the UI are not already selected. I found out this was intended to be a test for #1769 but I could not find there that this is required for this